server/utils/utils.test.js

```diff
@@ -95,3 +103,16 @@ describe('parseDate', () => {
     expect(parseDate('this is not a date', 'D MMM YYYY')).toEqual(null)
   })
 })
+
+describe('hash', () => {
+  it('should return hash', () => {
+    // result generated via https://approsto.com/sha-generator/
+    const result = stringToHash('abcABC')
+    expect(result).toEqual('XVnsPWdyGrzz4FzZhZQ1s2agjqxtfKqd3tNx1JC1tM8=')
+  })
+
+  it('should verify hash', () => {
+    const result = isHashedString('XVnsPWdyGrzz4FzZhZQ1s2agjqxtfKqd3tNx1JC1tM8=', 'abcABC')
+    expect(result).toBe(true)
```

Comment:
probably also want to test negative cases

Reply:
yes